dex/testing/loadbot/loadbot.go

```diff
@@ -101,6 +101,8 @@ var (
 
 	processesMtx = sync.Mutex{}
 	processes    = []*process{}
+
+	zecSendMtx = sync.Mutex{}
```

Comment:
Since `zecSendMtx` is not guarding a field, it could use a comment like `// prevents sending funds too soon after mining a block...` or something.

Reply:
Added a comment.